src/sdk/resources/auto_detect_compilers.xrc

```diff
@@ -65,14 +95,12 @@
 				<border>8</border>
 			</object>
 			<object class="sizeritem">
-				<object class="wxStdDialogButtonSizer">
-					<object class="button">
-						<object class="wxButton" name="wxID_OK">
-							<label></label>
-						</object>
+				<object class="wxBoxSizer">
```

Comment:
Removing the std-button-sizer is unacceptable it was added on purpose. It should span the whole bottom row of the dialog.

Reply:
See ticket 1110, when loading xrc in wxSmith wsWidget throws a debug alert with the wxStdDialogButtonSizer and this fixed the alert. So do you still want me to revert the change?

The debug alerts details are:
[Window Title]
wxWidgets Debug Alert

[Main Instruction]
A debugging check in this application has failed.

[Content]
../../src/common/wincmn.cpp(2490): assert ""!m_containingSizer"" failed in SetContainingSizer(): Adding a window already in a sizer, detach it first!

[ ] Don't show this dialog again  [Stop] [Continue]

Reply:
Who is the wxwidget XRC / wxsmith expert who can help with resolving this as I do not want to revert it and cause people headaches with the assert occurring.

Reply:
No idea. Probably there is none. Miguel Gimenez has made some changes to wxSmith recently.

But even if you refuse to revert this change, I'll revert it when I decide to push the changes to SVN.
The issue about the sizer is unrelated and it should be resolved separately. Doing work and then reverting it doesn't help!

Reply:
Due to this issue, this PR will not be able to be resolved until at least the middle of September as this is when Miguel Gimenez may have some time to look at the wxStdDialogButtonSizer issue.

Reply:
Really, you'll block work on this due to one unrelated minor issue? Keep in mind that I'll not look at any of your other PRs until this one is resolved. I don't have the capacity at the moment.

Reply:
Yes.